FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1054,6 +1057,210 @@ paths:
                 $ref: '#/components/schemas/Response'
       security:
       - OAuth2: [fraihmwork.component.read, fraihmwork.component.write, fraihmwork.component.admin]
+        
+  /blocklist:
+    get:
+      tags:
+      - Blocklist
+      summary: Requests the current blocklist
+      description: This call requests the list of all components and systems that are currently being blocklisted
+      operationId: getBlocklistedComponents
+      responses:
+        200:
+          description: Returned blocklist information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Blocklist'
+        401:
+          description: Not authenticated
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        406:
+          description: Not acceptable - only responds in application/json format
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - OAuth2: [fraihmwork.blocklist.read]
```

Comment:
I think fraihmwork.blocklist.manage should probably be able to read the blocklist, we moved towards this in the rest of the endpoints in this spec.